### PR TITLE
Dotnet masterclass week1

### PR DESCRIPTION
**Pull request by zmzade**

### CODE REVIEW

File: week1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    foreach (var vowel in input.ToLower())
+    {
+        if (vowels.Contains(vowel)) count++;
+    }
+    return count;
+});
+
+//4. Create function Fibonacci that returns N'th element of Fibonacci sequence
+//int n = 6 => Nth fibonacci number is 8
+app.MapGet("/fib", () =>
```
Comment (Vertygo) [inline week1/Program.cs]: You could also have a query parameter here to parse and output values based on it.

`app.MapGet("/fib", (int n) =>`

File: week1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+
+app.MapGet("/split", () =>
+{
+    int[] input = new[] { 10, 2, 5, 7, 2, 3, 5, 7 };
+    if (input.Length % 2 != 0) System.Console.WriteLine("the number of items should be even in array");
+
+    var mid = input.Length / 2;
+    var firstArr = input[..mid];
+    var secondArr = input[mid..];
```
Comment (Vertygo) [inline week1/Program.cs:81]: I see that you have found range operators. 👍

**Review by Vertygo — APPROVED** [review]

File: week1/Program.cs (hunk 1 above)
Comment (zmzade) [reply week1/Program.cs]: Thank you, good point. I add the query parameter and used also `TryParse` method in Fibonacci. I wanted to write the fibonacci as a recursive method but I donot know how to add classes and methods here.

Comment (Vertygo) [reply week1/Program.cs]: We still need to cover classes and methods.

**Review by Vertygo — APPROVED** [review]

**Pull request merged**